contrib/spdlog/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "spdlog"
+pkgver = "1.13.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DSPDLOG_FMT_EXTERNAL=ON",
+    "-DSPDLOG_BUILD_SHARED=ON",
+    "-DSPDLOG_BUILD_EXAMPLE=ON",
+]
+hostmakedepends = ["cmake", "fmt-devel", "ninja", "pkgconf"]
+pkgdesc = "Fast C++ logging library"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "MIT"
+url = "https://github.com/gabime/spdlog"
+source = (
+    f"https://github.com/gabime/{pkgname}/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "534f2ee1a4dcbeb22249856edfb2be76a1cf4f708a20b0ac2ed090ee24cfdbc9"
+# no checks
+options = ["!check"]
```

Comment:
it does have tests

Reply:
(you need to pass `-DSPDLOG_BUILD_TESTS=ON` to do that obviously)

Reply:
it requires libsystemd for tests

Reply:
libsystemd exists, it's in elogind-devel

..but it's also not required at all? quite literally, it's optional, and when not found it just skips the test_systemd.cpp one.